scripts/fmt.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Function to search directories recursively
+search_dirs() {
+    for dir in "$1"/*; do
+        if [ -d "${dir}" ]; then
+            if [ -f "${dir}/Acornfile" ]; then
```

Comment:
I think this will only find Acornfiles that are exactly named `Acornfile`, but there are some with other names (like `mysql.Acornfile`).

Not 100% sure if a wildcard would work for this, but it is worth trying:

```suggestion
            if [ -f "${dir}/*Acornfile" ]; then
```

Reply:
It's also possible for suffixes Acornfile.test etc.

Reply:
The script now handles prefixes and suffixes in the latest commit.